python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -52,12 +58,7 @@ class StandardAutoscaler:
     configure the right AWS/Cloud roles automatically.
 
     StandardAutoscaler's `update` method is periodically called by `monitor.py`
-    to add and remove nodes as necessary. Currently, load-based autoscaling is
-    not implemented, so all this class does is try to maintain a constant
-    cluster size.
-
-    StandardAutoscaler is also used to bootstrap clusters (by adding workers
-    until the cluster size that can handle the resource demand is met).
+    to add and remove nodes as necessary.
```

Comment:
Can you add elaborate more here?
At least mention that it autoscales based on demand and terminates when nodes are idle, etc.